Formula/bullet.rb

```diff
@@ -43,15 +43,21 @@ def install
     args_static = args.dup << "-DBUILD_SHARED_LIBS=OFF"
     args_static << "-DBUILD_BULLET2_DEMOS=OFF" if build.without? "demo"
 
-    system "cmake", ".", *args_shared
-    system "make", "install"
+    mkdir "build" do
+      system "cmake", "..", *args_shared
+      system "make", "install"
 
-    system "make", "clean"
```

Comment:
I'm not sure it does here. It doesn't remove the generated files from the `cmake` step; only the subsequent `make` step. And since we're doing another `cmake` for the second step, I'm afraid that cached values from the original `cmake` might be contaminating the second build. `cmake` is designed for iterative, interactive use, so values persist across calls to it.

Reply:
We do the same thing in `rocksdb` and `snappy` without any problems I'm aware of.

Reply:
Is there a reason not to simply use 2 separate build trees? Perhaps `build_shared` and `build_static`?

Reply:
I'll revert to `make clean` and fiddle with the cmake options to get it working.

I tried the separate `build_shared` and `build_static` originally, and it ended up with more lines and a little ugly, so I figured maintainers wouldn't like it. Works fine though.

Reply:
Amended.